src/Workspaces/Core/Portable/Workspace/Solution/TextLoader.cs

```diff
@@ -17,15 +17,23 @@ namespace Microsoft.CodeAnalysis
     /// </summary>
     public abstract class TextLoader
     {
+        private protected static readonly Workspace DummyWorkspace = new ImmutableWorkspace(Host.Mef.MefHostServices.DefaultHost, WorkspaceKind.Custom);
+        private static readonly DocumentId _dummyDocumentId = DocumentId.CreateNewId(ProjectId.CreateNewId("dummy"));
+
         private const double MaxDelaySecs = 1.0;
         private const int MaxRetries = 5;
         internal static readonly TimeSpan RetryDelay = TimeSpan.FromSeconds(MaxDelaySecs / MaxRetries);
 
         internal virtual string? FilePath => null;
 
+        internal Task<TextAndVersion> LoadTextAndVersionAsync(CancellationToken cancellationToken)
+            => LoadTextAndVersionAsync(DummyWorkspace, _dummyDocumentId, cancellationToken);
+
         /// <summary>
         /// Load a text and a version of the document.
         /// </summary>
+        /// <param name="workspace">Obsolete. Default, immutable <see cref="Workspace"/> instance.</param>
+        /// <param name="documentId">Obsolete. Dummy <see cref="DocumentId"/>.</param>
```

Comment:
slightly confused. t hese seem to refer to parameters that don't exist.

Reply:
LoadTextAndVersionAsync does have the parameters. Is github view misleading?

Reply:
Ah.  I was seeing this:

![image](https://user-images.githubusercontent.com/4564579/186957981-65064397-1348-45a0-a5d4-fff25f708958.png)

Didn't realize there was a collapsed section in between :)